src/table/components/head/HeadCellMenu.tsx

```diff
@@ -1,23 +1,35 @@
 import React, { useRef, useState, useMemo } from 'react';
 import Menu from '@mui/material/Menu';
-import More from '@qlik-trial/sprout/icons/react/More';
-import Search from '@qlik-trial/sprout/icons/react/Search';
+import More from '@qlik-trial/sprout/icons/More';
+import Search from '@qlik-trial/sprout/icons/Search';
 
 import { useContextSelector, TableContext } from '../../context';
 import { HeadCellMenuProps, MenuItemGroup } from '../../types';
-import { StyledMenuIconButton, NebulaListBox } from './styles';
-import { ListBoxWrapper, ListBoxWrapperRenderProps } from './ListBoxWrapper';
+import { StyledMenuIconButton } from './styles';
 import MenuItems from './MenuItems';
 
 export default function HeadCellMenu({ columnIndex, isDimension, tabIndex }: HeadCellMenuProps) {
-  const { translator } = useContextSelector(TableContext, (value) => value.baseProps);
+  const { translator, embed, layout } = useContextSelector(TableContext, (value) => value.baseProps);
   const [openMenuDropdown, setOpenMenuDropdown] = useState(false);
-  const [openListboxDropdown, setOpenListboxDropdown] = useState(false);
-  const anchorRef = useRef<HTMLButtonElement>(null);
+  const anchorRef = useRef<HTMLDivElement>(null);
+  const listboxRef = useRef<HTMLDivElement>(null);
+  const isMasterDimension = layout.qHyperCube.qDimensionInfo[columnIndex]?.qLibraryId;
+
+  const embedListbox = () => {
+    if (!layout || !embed) return;
+    const fieldId = layout.qHyperCube.qDimensionInfo[columnIndex]?.qFallbackTitle;
+    if (!fieldId) return;
+    // @ts-ignore `__DO_NOT_USE__` api does not require type check
+    // eslint-disable-next-line
```

Comment:
I think this is a nice pattern to do this https://github.com/qlik-oss/sn-table/blob/main/src/table/utils/selections-utils.ts#L41

Reply:
You mean adding a TODO before @ts-ignore description? 
we also need lint ignore because of using __ before api name that we don't have control over it

Reply:
yeah